src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,51 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int DEFAULT_ITEMS_NUMBER = 10;
+
+    private K[] keys;
+
+    private V[] values;
+    private int size;
+
+    public StorageImpl() {
+        keys = (K[]) new Object[DEFAULT_ITEMS_NUMBER];
+        values = (V[]) new Object[DEFAULT_ITEMS_NUMBER];
+    }
+
+    public K[] getKeys() {
+        return keys;
+    }
+
+    public V[] getValues() {
+        return values;
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < size; i++) {
+            if (keys[i] == key || keys[i] != null && keys[i].equals(key)) {
+                values[i] = value;
+                size--;
```

Comment:
```suggestion
                return;
```
In this case, you need to prevent further execution, you've done everything you intended to. This is void method, so we can simply call `return; `